wsgi.py

```diff
@@ -1,16 +1,30 @@
+import os
+import re
+import cgi
+import random
+import mimetypes
+
 import wsgiref.validate
 
 from utils import parse_http_x_www_form_urlencoded_post_data, \
     get_first_element, parse_http_get_data, parse_http_headers, \
     parse_http_content_type, parse_http_uri
 
+
+CWD = os.path.dirname(os.path.abspath(__file__))
+
 DEBUG = True
 STATIC_URL = '/static/'
-STATIC_ROOT = 'data'
+STATIC_ROOT = os.path.join(CWD, 'data')
+
+MESSAGE_PATTERN = '<p class="name">{0}</p><p class="message">{1}</p>'
```

Comment:
If you separate pattern definition and pattern usage You must use more readable pattern definition.
Example:
MESSAGE_PATTERN = '<p class="name">{name}</p><p class="message">{message}</p>'
MESSAGE_PATTERN.format(name=name, message=message)